<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDlg</class>
 <widget class="QDialog" name="CreateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_reset">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_strg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Strategy</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_Dir">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_qty">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_offset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_log">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_price">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_exch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>177</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exchange</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_strg">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_rmk1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_rmk1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>112</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remark1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rmk2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>112</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remark2</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_rmk2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_symbol">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_dir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>177</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_log">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>112</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log Infor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_order">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_exch">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_symbol">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Symbol</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_offset">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_qty">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>239</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ctp">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect CTP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
